controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -999,6 +1000,12 @@ func saveHostProvisioningSettings(host *metal3v1alpha1.BareMetalHost) (dirty boo
 		}
```

Comment:
@dtantsur Can we continue the [question](https://github.com/metal3-io/baremetal-operator/pull/292/files#r553245789) left over in PR292 here?